src/main/java/es/tid/pce/pcep/objects/tlvs/SRPolicyName.java

```diff
@@ -0,0 +1,119 @@
+package es.tid.pce.pcep.objects.tlvs;
+
+import java.util.Objects;
+
+import es.tid.pce.pcep.objects.MalformedPCEPObjectException;
+import es.tid.pce.pcep.objects.ObjectParameters;
+
+/*
+ * SR Policy Name TLV
+ * @author Luis Cepeda Martínez
+ */
+
+public class SRPolicyName extends PCEPTLV{
+	/**
+	 * 0                   1                   2                   3
+       0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+      |             Type              |             Length            |
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+      |                                                               |
+      ~                       SR Policy Name                          ~
+      |                                                               |
+      +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+                  Figure: The SRPOLICY-POL-NAME TLV format
+                  
+                  
+	 */
+
+	private String policyName;
+	
+	
+	public SRPolicyName(){
+		this.TLVType=56;
```

Comment:
```suggestion
		this.TLVType=ObjectParameters.PCEP_TLV_SRPOLICY_POL_NAME;
```

Create the entry in ObjectParameters with the type